scripts/parse.py

```diff
@@ -515,46 +534,58 @@ def indentandrawoffset2globaloffset(indent: bool, rawoffset: int, offsetlist: Li
     raise RuntimeError('There\'s a bug in this code. This statement should not have been reached.')
 
 
-if __name__ == '__main__':
-    if len(sys.argv) != 2:
-        print('Usage: %s <captureddatahexdump>' % sys.argv[0])
-        print('')
-        print('This program will parse a wireshark hexdump of communication')
-        print('between the TA client and the login server (port 9000) and')
-        print('dump it into a text file')
-        exit(0)
-
-    infilename = sys.argv[1]
-
-    with open(infilename, 'rt') as infile:
-        with open(infilename + '_parsed.txt', 'wt') as outfile:
+@click.command()
+@click.option('--file', '-f', prompt='File to parse', help='hexdump to parse')
+@click.option('--disable-id-annotations', is_flag=True,
```

Comment:
I think it's potentially useful in general for looking through the script for certain things - particularly if we have a _lot_ of known values it can add a lot of clutter.

The other thing is I've started working on parsing the output produced by parse.py to generate a list of constants from the purchase section, which should give us the complete list of (at least) weapon / item IDs purely from the network protocol.

Reply:
Ok. I just want to watch out for having too many options that nobody ever uses. Keep them for now.